src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -7834,11 +7834,12 @@ instruct membar_release() %{
   ins_cost(VOLATILE_REF_COST);
 
   format %{ "membar_release\n\t"
-            "dmb ish" %}
+            "dmb ishst\n\tdmb ishld" %}
 
   ins_encode %{
     __ block_comment("membar_release");
-    __ membar(Assembler::LoadStore|Assembler::StoreStore);
+    __ membar(Assembler::StoreStore);
```

Comment:
Do we need to respect `AlwaysMergeDMB`here?

Reply:
Yes, usually they can be merged in macroAssembler. but  it can help to reduce the possibility of unmerged case. Thanks to point it.

Reply:
I checked code again. They will be merged if enable AlwaysMergeDMB. So we can skip the check.

Reply:
Add a comment:

`// These will be merged if AlwaysMergeDMB is enabled.`

Reply:
Comment added.